commec/tools/blastn_handler.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021-2024 International Biosecurity and Biosafety Initiative for Science
+"""
+Defines the `Input and Output Screen Parameters` class, and associated dataclasses.
+"""
+
+import subprocess
+from commec.tools.blast_tools import BlastHandler
+from commec.tools.database_handler import DatabaseVersion
+
+class BlastNHandler(BlastHandler):
+    """ A Database handler specifically for use with BlastX files for commec screening. 
+    Allows for full customization of any of the callable blast flags. A better implementation
+    may be achieved using import blast"""
```

Comment:
Sorry, code thoughts creeping their way into comments, I believe this was removed in a recent tidying commit. Yes I was referring to biopython when considering best practices to call CLIs, but since decided to implement the search_handler CLI system we use now.

Reply:
Slight alterations to the docstrings for blastn , diamond and Blastx to reflect the handlers.